flask/app/__init__.py

```diff
@@ -16,32 +23,47 @@ def create_app(config):
 
     # Create the application object
     app = Flask(__name__)
-
     app.config.from_object(config)
 
-    if config.SQLALCHEMY_LOG:
-        logging.basicConfig()
-        logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
-        logging.getLogger("sqlalchemy.pool").setLevel(logging.INFO)
+    register_extensions(app)
+    register_commands(app)
+    register_blueprints(app)
+    config_logger(app)
```

Comment:
Logger should be registered first before the extensions are registered to match the order before.

Reply:
fixed